src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/person/domain/OppfolgingstilfelleDag.kt

```diff
@@ -76,24 +81,34 @@ private fun List<OppfolgingstilfelleDag>.findLastPriorityOppfolgingstilfelleBit(
         it.priorityOppfolgingstilfelleBit != null
     }.priorityOppfolgingstilfelleBit
 
-fun List<OppfolgingstilfelleDag>.gradertAtTilfelleEnd() =
-    this.findLastPriorityOppfolgingstilfelleBit()?.isGradert() ?: false
+fun List<OppfolgingstilfelleDag>.allGradert() =
+    this.all { dag -> dag.priorityOppfolgingstilfelleBit?.isGradert() ?: false }
+
+fun List<OppfolgingstilfelleDag>.findDag(date: LocalDate) =
+    this.find { oppfolgingstilfelleDag -> oppfolgingstilfelleDag.dag == date }
 
-fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle(): Oppfolgingstilfelle {
+fun List<OppfolgingstilfelleDag>.toOppfolgingstilfelle(
+    oppfolgingstilfelleDagerPerVirksomhet: Map<String?, List<OppfolgingstilfelleDag>>,
```

Comment:
Siden nøkkelen til map'en kan være null, så blir kanskje de bitene som mangler virksomhetsnummer (feks biter som er BEKREFTET) samlet for denne?

Reply:
Stemmer at de blir samlet da ja - tenker du det blir feil (at det kan være forskjellige arbeidsforhold) ?

Reply:
Tror dette kan funke bra, men blir lurt med noen tester som faktisk viser at det oppfører seg som forventet.

Reply:
Da samler kanskje noen som ikke hører sammen da, men det er kanskje ikke noe vi får gjort med det? 🤔 Man kan jo i teorien både være selvstendig næringsdrivende og frilans (selv om det sikkert ikke er så vanlig) 🙃